src/main/app/form/validation.ts

```diff
@@ -106,8 +108,20 @@ export const isPhoneNoValid: Validator = value => {
   }
 };
 
+export const isEmailValid: Validator = value => {
+  if (!isValidEmail(value as string)) {
+    return 'invalid';
+  }
+};
+
 export const isFieldLetters: Validator = value => {
   if (!(value as string).match(/^[a-zA-Z][a-zA-Z\s]*$/)) {
     return 'invalid';
   }
 };
+
+export const doesNotKnowEmail = (value: Partial<Case>): string | void => {
+  if (value.respondentEmailAddress && value.doNotKnowRespondentEmailAddress === Checkbox.Checked) {
```

Comment:
i feel like, if the other validation was moved due to being specific to the content, this should also be moved.

Reply:
I guess you could say the same thing about  **isInvalidHelpWithFeesRef** too